Makefile.in

```diff
@@ -564,6 +572,10 @@ endif
 	@for f in $(BINDINGS); do \
 	  install -m 0644 $$f $(DESTDIR)$(includedir)/@APP_NAME_LC@ ; \
 	done
+	@for d in $(BINDINGS_DIR); do \
+	  install -d $(DESTDIR)$(includedir)/kodi/$$d; \
+	  (find xbmc/addons/kodi-addon-dev-kit/include/kodi/$$d -maxdepth 1 -not -type d -type f -exec install -m 0644 "{}" $(DESTDIR)$(includedir)/kodi/$$d \;) \
```

Comment:
I bring currently this back. About the "()", it was required about a conflict with the "\;" on end of find call.

Is there a alternative way about it?